Models/c172p.xml

```diff
@@ -6587,5 +6587,273 @@
         </axis>
     </animation>
     <!-- END MOD: Pontoon Gear -->
+    
+    <!-- Pitot tube cover "remove before flight" -->
+    <model>
+        <name>rbf-pitot</name>
+        <path>/Exterior/rbf/rbf.xml</path>
+        <offsets>
+            <x-m>-0.220</x-m>
+            <y-m>-1.525</y-m>
+            <z-m>0.464</z-m>
+        </offsets>
+    </model>
+    <animation>
+        <type>select</type>
+        <object-name>rbf-pitot</object-name>
+        <condition>
+            <property>sim/model/c172p/pitot-cover</property>
+        </condition>
+    </animation>
+    <animation>
+        <type>pick</type>
+        <object-name>rbf-pitot</object-name>
+        <action>
+            <button>0</button>
+            <repeatable>false</repeatable>
+            <binding>
+                <condition>
+                    <and>
+                        <property>sim/model/c172p/pitot-cover</property>
+                        <less-than>
+                            <property>velocities/groundspeed-kt</property>
+                            <value>5.0</value>
+                        </less-than>
+                    </and>   
+                </condition>
+                <command>property-assign</command>
+                <property>instrumentation/airspeed-indicator/serviceable</property>
+                <value>true</value>
+            </binding>
+            <binding>
+                <condition>
+                    <and>
+                        <property>sim/model/c172p/pitot-cover</property>
+                        <less-than>
+                            <property>velocities/groundspeed-kt</property>
+                            <value>5.0</value>
+                        </less-than>
+                    </and>   
+                </condition>
+                <command>property-assign</command>
+                <property>sim/model/c172p/pitot-cover</property>
+                <value>false</value>
+            </binding>
+        </action>
+        <hovered>
+            <binding>
+                <condition>
+                    <and>
+                        <property>sim/model/c172p/pitot-cover</property>
+                        <less-than>
+                            <property>velocities/groundspeed-kt</property>
+                            <value>5.0</value>
+                        </less-than>
+                    </and>   
+                </condition>
+                <command>set-tooltip</command>
+                <tooltip-id>remove-pitot-tube-cap</tooltip-id>
+                <label>Remove Pitot Tube Cap</label>
+            </binding>
+        </hovered>
+    </animation>
+    <animation>
+        <type>pick</type>
+        <object-name>PitotTube</object-name>
+        <action>
+            <button>0</button>
+            <repeatable>false</repeatable>
+            <binding>
+                <condition>
+                    <and>
```

Comment:
You have this logic here multiple times (in different bindings), so better put it in a `<logic>` element then.